### PR TITLE
Misc Botany changes.

### PR DESCRIPTION
**Pull request by Firecage**
1.This gives names to the different cannabis leaves, since at the moment, lets say you put 1 cannabis leaf, 1 omega cannabis leaf, and 1 white cannabis leaf into the smart fridge, it will show that there's 3 cannabis leaves in it, since it will bundle them up. This fixes that, and actually gives them names.

2.This PR also enabled you to add weapon/grown items to the Smart Fridge. This includes Nettles(and Death nettles), snapcorn, sunflowers(and moon flowers and novaflowers), towercap logs(and steelcap logs). As a side effect, you will also be able to add banana peels and corncobs to the smart fridge since they are also weapon/grown. But I don't think that is to bad.

3.This PR also gives new sprites to the Cannabis seed packs, firstly, so the different types looks different. Secondly. So the seed packs are consistent with those we already have.

enk5ibp

### CODE REVIEW

**Comment by Supermichael777** [comment]
now all we need is a glove plant.

File: code/modules/hydroponics/grown/cannabis.dm [hunk 1]
```diff
@@ -2,7 +2,6 @@
 /obj/item/seeds/cannabis
 	name = "pack of cannabis seeds"
 	desc = "Taxable."
-	icon = 'goon/icons/obj/hydroponics.dmi'
```
Comment (RemieRichards) [inline code/modules/hydroponics/grown/cannabis.dm]: No, goon icons must remain in the goon folder.

Comment (bobdobbington) [inline code/modules/hydroponics/grown/cannabis.dm]: I don't think the new icons are from goon? Could be wrong though.

Comment (RemieRichards) [inline code/modules/hydroponics/grown/cannabis.dm]: I'm pretty sure he mentioned in IRC earlier something about moving the icons to the main file.
which is a no-no for goon icons.

**Comment by jughu** [comment]
IT is goons

File: code/modules/hydroponics/grown/cannabis.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/hydroponics/grown/cannabis.dm]: These are just the seed packets remie, which he remade

Comment (Firecage) [inline code/modules/hydroponics/grown/cannabis.dm]: No Remie, as I said in the PR itself aswell. I made new sprites for the seed packets for the cannibas plants. Said sprites are in the normal seeds sprites file. Heck, the seeds are even pictured.

**Comment by optimumtact** [comment]
Well which is it, goons or not goons?

**Comment by Iamgoofball** [comment]
@optimumtact It's fine.

He replaced goon sprites with non-goon sprites. So, he moved his new non-goon sprites to the normal folders.

**Comment by Firecage** [comment]
Oranges really needs glasses so he can read what the PR says.

> This PR also gives new sprites to the Cannabis seed packs

**Comment by optimumtact** [comment]
explain this

I was responding to this

**Comment by Firecage** [comment]
You know, I don't know what Jughu was talking about either, or who he is.

**Comment by jughu** [comment]
The old one i meant

**Pull request merged**